README.md

```diff
@@ -114,6 +116,46 @@ To stop the containers run:
 
 ## Application Concepts
 
+## Usage
```

Comment:
WDYT about extracting this to another markdown file? As we did with [contributing](https://github.com/comarev/comarev/blob/master/CONTRIBUTING.md) for example

Reply:
Sure! Where should I link this usage section?

Reply:
I think it will fit well right before the `Application Concepts` section